azure-auth-helper/src/test/java/com/microsoft/azure/auth/AzureAuthHelperTest.java

```diff
@@ -61,12 +61,8 @@ public void testRefreshTokenInvalidParameter() throws Exception {
 
     @Test
     public void tetGetAzureSecretFile() throws Exception {
-        File azureSecretFile = AzureAuthHelper.getAzureSecretFile();
-        assertEquals(Paths.get(System.getProperty(Constants.USER_HOME_KEY), ".azure", "azure-secret.json").toString(),
-                azureSecretFile.getAbsolutePath());
-        System.setProperty(Constants.AZURE_HOME_KEY, "test_dir");
-        azureSecretFile = AzureAuthHelper.getAzureSecretFile();
-        assertEquals(Paths.get("test_dir", "azure-secret.json").toFile().getAbsolutePath(),
+        final File azureSecretFile = AzureAuthHelper.getAzureSecretFile();
+        assertEquals(Paths.get(System.getProperty(Constants.USER_HOME), ".azure", "azure-secret.json").toString(),
```

Comment:
change to `Constants.AZURE_FOLDER` `Constants.AZURE_SECRET_FILE `

Reply:
this is a test case, we are free to use actual constant values rather than Constants.AZURE_SECRET_FILE